### PR TITLE
Add glibc-supported python demo

### PR DESCRIPTION
**Pull request by ClawSeven**
Though users prefer to install python on Ubuntu using apt, the build-in python is position-dependent and incompatible with occlum. However, the python provided from conda is position-independent, we choose miniconda as python installation tool, which is a free minimal installer for conda. The users can load the required python packages more feasible using conda,

### CODE REVIEW

File: demos/python/python_glibc/install_python_using_conda.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" >/dev/null 2>&1 && pwd )"
+
+# 1. Init occlum workspace
```
Comment (guzongmin) [inline demos/python/python_glibc/install_python_using_conda.sh:5]: Init a new occlum instance with "occlum new".

File: .github/workflows/demo_test.yml [hunk 2]
```diff
@@ -375,10 +375,39 @@ jobs:
       run: docker exec python_support_test bash -c "source /opt/intel/sgxsdk/environment; cd /root/occlum; OCCLUM_RELEASE_BUILD=1 make install"
```
[the remark this answers is not in the document]
Comment (ClawSeven) [reply .github/workflows/demo_test.yml:375]: Done.

File: demos/python/python_glibc/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## How to Run
+
+This tutorial is written under the assumption that you have Docker installed and use Occlum in a Docker container.
+
+Occlum is compatible with glibc-supported Python, we employ miniconda as python installation tool. You can import any python dependencies using conda. Here, miniconda is automatically installed by install_python_using_conda.sh script, the required python and related dependencies for this project are also loaded by this script. Here, we take occlum/occlum:0.22.0-ubuntu18.04 as example.
+
+Step 1 (on the host): Start an Occlum container
+```
+docker pull occlum/occlum:0.22.0-ubuntu18.04
+docker run -it --privileged --name=pythonDemo --device /dev/sgx/enclave occlum/occlum:0.22.0-ubuntu18.04 bash
```
[the remark this answers is not in the document]
Comment (ClawSeven) [reply demos/python/python_glibc/README.md:18]: Not necessary, Done.

File: demos/python/python_glibc/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+Step 2 (on the host): Download miniconda and install python to prefix position.
+```
+cd /root/occlum/demos/python/python_glibc
+bash ./install_python_using_conda.sh
```
[the remark this answers is not in the document]
Comment (ClawSeven) [reply demos/python/python_glibc/README.md:24]: Done

File: demos/python/python_glibc/install_python_using_conda.sh (hunk 1 above)
Comment (ClawSeven) [reply demos/python/python_glibc/install_python_using_conda.sh:5]: Done

**Review by guzongmin — COMMENTED** [review]
The demos/README.md should be updated.

**Comment by ClawSeven** [comment]
The demos/README.md have been updated.

**Review by guzongmin — APPROVED** [review]
LGTM

**Pull request merged**